rasa/nlu/tokenizers/jieba_tokenizer.py

```diff
@@ -108,12 +111,21 @@ def preprocess_text(self, text, attribute):
         else:
             return text
 
-    def tokenize(self, text: Text, attribute=MESSAGE_TEXT_ATTRIBUTE) -> List[Token]:
+    def tokenize(
+        self, text: Text, attribute: Text = MESSAGE_TEXT_ATTRIBUTE
+    ) -> List[Token]:
         import jieba
 
         text = self.preprocess_text(text, attribute)
         tokenized = jieba.tokenize(text)
         tokens = [Token(word, start) for (word, start, end) in tokenized]
+
+        if (
+            attribute in [MESSAGE_RESPONSE_ATTRIBUTE, MESSAGE_TEXT_ATTRIBUTE]
+            and self.add_cls_token
+        ):
+            tokens.append(Token(CLS_TOKEN, len(text) + 1))
```

Comment:
maybe add the comment, why `+1`, I guess it is because of a space between last word and `__cls__` token

Reply:
yes, will add a comment